pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!--Please fill in the **bold** fields, and tick all applicable boxes.-->
+
+Added **your URL here**.
```

Comment:
I don't see why this should be a comment. I think if the PR contains a clickable link to the list, that adds convenience for everybody. Instead of inspecting the commit, copy-pasting the link into the address bar, we can now just click it and navigate right away.

That is also how I interpret the contributing guidelines

> The body of your commit message should contain a link to the repository.